.github/workflows/distcheck.yaml

```diff
@@ -118,16 +118,16 @@ jobs:
           fi
 
       - name: Call `make`
-        run: make
+        run: make V=1
```

Comment:
I'm fine with this approach, but might be easier to just run configure with `--disable-silent-rules`.

Reply:
Wasn't aware that is an option -- only knew about `V=1`. I think we could go either way.